tests/jax/components/training/losses_test.py

```diff
@@ -239,7 +239,6 @@ def test_mapg_loss_separate_networks(
     low_loss_critic = low_agent_0_critic_loss["loss_critic"]
 
     assert jnp.isclose(loss_entropy, -0.47500002)
-    assert loss_critic == 4.5
```

Comment:
Can you maybe add a test for the Huber loss? Instead of needing to remove this assert?

Reply:
I've added tests for the separated functions and readded the assert.

Reply:
Thanks :)